target/linux/ipq40xx/files-5.4/arch/arm/boot/dts/qcom-ipq4018-m018u0-s1.dts

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qcom-ipq4018-m018u0.dtsi"
+
+/ {
+	model = "Corewav M018U0 SOM S1";
+	compatible = "corewav,m018u0-s1";
+
+	memory {
+		device_type = "memory";
+		reg = <0x80000000 0x10000000>;
+	};
+};
+
+&blsp1_spi1 {
+	mx25l25635f@0 {
```

Comment:
@neverhover Can you please change the node name to more generic name `flash@0` and try if it boots for you?